lib/hardware_services/inventory_drivers/MockInventoryDriver.py

```diff
@@ -151,14 +151,47 @@ def load_data(self, quadsinstance, force):
             quadsinstance.logger.error(ex)
             exit(1)
 
-    def write_data(self, quadsinstance, doexit = True):
-        quadsinstance.quads_write_data_(doexit)
+    def write_data(self, quadsinstance, doexit):
```

Comment:
What is different about these changes and the quads native changes?  They seem identical.  Is that supposed to be the case?